src/main/java/frc/robot/commands/RaiseArms.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.subsystems.Climber;
+import edu.wpi.first.wpilibj.Timer;
+
+public class RaiseArms extends Command {
+  private final Climber climber;
+  public RaiseArms(Climber climber) {
+    this.climber = climber;
+    addRequirements(climber);
+  }
+  private double startTime;
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    startTime = Timer.getFPGATimestamp();
+    climber.raise(true);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    if (Timer.getFPGATimestamp() - startTime > Constants.RAISING_TIMEOUT) {
+      return true;
+    }
+    return false;
```

Comment:
Could be simpler written like so:
```suggestion
    return Timer.getFPGATimestamp() - startTime > Constants.RAISING_TIMEOUT;
```